vms/platformvm/txs/fee/static_calculator.go

```diff
@@ -119,3 +119,8 @@ func (c *staticVisitor) ExportTx(*txs.ExportTx) error {
 	c.fee = c.config.TxFee
 	return nil
 }
+
+func (c *staticVisitor) ConvertSubnetTx(*txs.ConvertSubnetTx) error {
+	c.fee = c.config.TxFee
```

Comment:
This will never be used because this tx will only be valid in Etna which is switching over to the dynamic calculator. Is this the right placeholder for interface compliance? cc @StephenButtolph